cves/CVE-2018-7537.yml

```diff
@@ -260,7 +289,9 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    This seems like it was a testing mistake to me. I think that the vunerability could have been found
```

Comment:
I feel as though this needs a little more analysis, perhaps expand on what specific types of unit tests would have been useful.